web/src/components/dropdowns/query-options-dropdown.tsx

```diff
@@ -203,6 +230,51 @@ export const QueryOptionsPanel: React.FC<QueryOptionsDropdownProps> = ({
           </div>
         ))}
       </div>
+      <div className="pf-c-select__menu-group">
+        <Tooltip
+          content={t(
+            // eslint-disable-next-line max-len
+            'Records to show from dropped, sent or mix of both.'
```

Comment:
I think it deserves a more detailed explanation, as it doesn't seem trivial to understand all the options.

For instance I didn't immediately understand what was the difference between Dropped and Contains drops, especially when it happens to show the exact same thing (which seems to be often the case in the flow table). So, maybe saying something like "Dropped" shows the flows that are 100% dropped whereas "Contains drops" shows the flows having at least one packet dropped.

Also, when reading "Packet loss" > "All" , it can be ambiguous as we may understand that it's going to show all drops. It's probably the title "Packet loss" that is misleading, maybe it should be "Drops filter" ? And options like:
- Fully dropped
- Contains drops
- No drops
- All (or any?)

cc @stleerh any take on this?

Reply:
cc @Amoghrd too

Reply:
Here we go https://github.com/netobserv/network-observability-console-plugin/pull/324/commits/e5edc3039eb5f407d1593ac20ccd1dfbeaaeae67

![image](https://github.com/netobserv/network-observability-console-plugin/assets/91894519/76e34441-7709-4782-bedc-31a55d9a8ab6)